core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerBackend.scala

```diff
@@ -59,6 +59,10 @@ private[spark] class MesosSchedulerBackend(
 
   private[mesos] val mesosExecutorCores = sc.conf.getDouble("spark.mesos.mesosExecutor.cores", 1)
 
+  // Offer constraints
+  private[mesos] val slaveOfferConstraints =
```

Comment:
This can just be `private`, it doesn't need to be `private[mesos]`.  (`mesosExecutorCores` is `private[mesos]` so that it can be read in the tests, but this isn't read outside of this class.)